_episodes/01-basics.md

```diff
@@ -1,13 +1,20 @@
 ---
-layout: page
-title: Testing
-subtitle: Basics of Testing
-minutes: 5
+title: Basics of Testing
+teaching: 5
+exercises: 0
+questions:
+- "Why would you not want to test?"
+objectives:
+- "Understand the place of testing in a scientific workflow."
+- "Understand that testing has many forms."
+keypoints:
+- "Tests compare that the result observed from running code is the same as what was expected ahead of time."
+- "Tests should be written at the same time as the code they are testing is written."
```

Comment:
Same concern here. I don't necessarily advocate a single time for writing tests.